README.md

```diff
@@ -121,9 +61,23 @@ export PACKAGE_FILE=flapjack_1.1.0~+20141003112645-master-trusty-1_amd64.deb
 bundle exec rake build
 ```
 
+### Testing Packages
+
+The test task starts up a docker instance with a pristine copy of the distribution, installs the package given, and runs Serverspec against it.
+
+The following environment variable is required:
+
+- `PACKAGE_FILE` - the filename of the package file you've just built and want to publish.  This is assumed to be in omnibus-flapjack/pkg (which is where the Rakefile build puts it)
```

Comment:
I'm pretty sure this will also download the package from packages.flapjack.io if it's not found in `omnibus-flapjack/pkg` - i believe I tested that recently.